src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.helpers.ts

```diff
@@ -194,7 +194,7 @@ export class DatasetFlowTableHelpers {
                 return "waiting for scheduled execution";
 
             default:
-                return "";
+                return "Unknown flow start condition";
```

Comment:
Similarly, I don't want to see any text in this case - this would scare the user

Reply:
Fixed